tests/test_json.py

```diff
@@ -11,13 +20,15 @@ def evaluate(ll):
         return ll
 
 
-def to_bytes(value):
+def normalize_input(value):
     if isinstance(value, str):
-        return utils.str_to_bytes(value)
+        return codecs.encode(value, 'utf8')
```

Comment:
Not important during a test, but since we're hyper-aware of performance right now, maybe it's worth mentioning that `value.encode('utf8')` is more than 2x faster:

```py
In [156]: profile(lambda: codecs.encode("Now we see the violence inherent in the system", 'utf8'), runs=10000000)
took 4.757038593292236s to run 10000000 times. 0.47570385932922366us per run

In [157]: profile(lambda: "Now we see the violence inherent in the system".encode('utf8'), runs=10000000)
took 2.0106379985809326s to run 10000000 times. 0.20106379985809325us per run
```